added emoji to PostMessageIntent

TheDreamSaver opened the pull request:
Now you can say, send "I'm feeling happy emoji heart" to general.

TheDreamSaver committed:
added emoji to PostMessageIntent

Now you can say, send "I'm feeling happy emoji heart" to general.

PrajvalRaval reviewed (changes requested):
Add a sample utterance to our SampleUtterance file and along with cheatsheet and 3 emoji limit in your PR :thumbsup:

acerbisgianluca commented on models/en-IN.json:
```diff
@@ -61,14 +61,29 @@
                 },
                 {
-                    "name": "GetLastMessageFromChannelIntent",
+                    "name": "GetLastMessageFromChannel",
```
Please change `GetLastMessageFromChannel` to `GetLastMessageFromChannelIntent` in `models`.

TheDreamSaver committed:
Added new slot values which work with Rocket Chat's core

Rocket chat uses https://raw.githubusercontent.com/joypixels/emojione-assets/master/emoji.json

Added them as slot values after cleaning the skin tone ones.
